constituencies_mapping/transform_constituencies.py

```diff
@@ -0,0 +1,26 @@
+import numpy as np
+import h5py
+import os
+
+def transform_2010_to_2024(weights_2010, file_path):
```

Comment:
In future, please add type hints to all functions (Claude can probably do this). Helps with VS Code editors!